### DISCUSSION TITLE
Fix Docker

### DISCUSSION THREAD

**Discussion by sudoskys** [LlmKira/Openaibot#317] [Announcements]
lib0.26.2

<hr /><em>This discussion was created from the release <a href='https://github.com/LlmKira/Openaibot/releases/tag/lib0.26.2'>Fix Docker</a>.</em>

**Comment by zolali** [comment]
hi
what is version of docker in buffer overflow server lab?and how I run docker-compose.yml file?when I write the command" "

**Reply by sudoskys** [reply]
We use
```sh
docker -v
Docker version 24.0.7, build afdd53b
```
Version:
https://github.com/LlmKira/Openaibot/releases/tag/nolib0.26.3_2

Docs: https://llmkira.github.io/Docs/en/guide/getting-started

````sh
apt install docker-compose
cd Openaibot

cp .env.exp .env && nano .env 
# Add your setting in .env

docker-compose -f docker-compose.yml up -d
````

**Reply by sudoskys** [reply]
Then use `docker logs llmbot` show logs

**Reply by sudoskys** [reply]
If crash, run `docker-compose -f docker-compose.yml up` under `Openaibot`, copy the error log output and paste here....(before do that ,delete private key/token)

**Reply by sudoskys** [reply]
If you do not install docker yet, run `curl -fsSL https://get.docker.com | bash -s docker` as root user.

**Comment by zolali** [comment]
Jasmin can you create an account in openAI? I am in Iran and it is
restricted for my country

**Comment by zolali** [comment]
I did all your commands but it doesn't work, is not better to remove it and
install again? Commands on seedlab is for which version?

**Reply by sudoskys** [reply]
When you are in Iran, I just cant reach u

**Reply by sudoskys** [reply]
I don't know what your situation is, maybe your system is too old
